app.rb

```diff
@@ -1,8 +1,27 @@
-require 'sinatra/base'
+require "sinatra/base"
+require "./lib/peep"
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  get "/test" do
+    "Test page"
+  end
+
+  get "/" do
+    "Chitter"
+  end
+
```

Comment:
You may want to include this in a separate erb file, so that this route isn't handling any "view" data.